users/pcoves/pcoves.c

```diff
@@ -0,0 +1,42 @@
+#include "pcoves.h"
+
+#ifdef RAINBOW_UNICORN_ENABLE
+#include "rainbowUnicorn.h"
+#endif
+
+#ifdef UNICODE_ENABLE
+#include "unicode.h"
+#endif
+
+#if SECRET_ENABLE
+#include "secret.h"
+#endif
+
+void eeconfig_init_keymap(void) { return; }
```

Comment:
This doesn't need a return, since it is typed as `void`.  

Also, if you want to be able to use this in your keymap.c files, this needs to be declared weak.  Eg: 
```suggestion
__attribute__((weak)) void eeconfig_init_keymap(void) { }
```